src/statement_generator.cpp

```diff
@@ -1123,6 +1139,7 @@ unique_ptr<ParsedExpression> StatementGenerator::GenerateSubquery() {
 
 	{
 		StatementGenerator child_generator(*this);
```

Comment:
Here too, the `child_generator.verification_enabled = verification_enabled` can happen in the constructor